nixos/modules/services/x11/display-managers/default.nix

```
@@ -321,14 +321,15 @@ in
         execCmd = mkOption {
           type = types.str;
           example = literalExample ''
-            "''${pkgs.lightdm}/bin/lightdm"
+            "''${pkgs.slim}/bin/slim"
```

Let’s keep it on the down low.

```suggestion
            "''${pkgs.lightdm}/bin/lightdm"
```